periodic/periodic.go

```diff
@@ -23,20 +23,23 @@
 package periodic // import "istio.io/fortio/periodic"
 
 import (
-	"errors"
 	"fmt"
+	"io"
 	"os"
 	"os/signal"
 	"runtime"
-	"strconv"
-	"strings"
 	"sync"
 	"time"
 
 	"istio.io/fortio/log"
 	"istio.io/fortio/stats"
 )
 
+const (
```

Comment:
Just wondering whether `fortio_main.go` is better place to define the version

Reply:
the main isn't part of the library so that wouldn't work (circular dependencies) - I considered renaming the periodic package to fortio - not sure